synfig-studio/src/gui/dialogs/canvasresize.cpp

```diff
@@ -0,0 +1,229 @@
+/* === S Y N F I G ========================================================= */
+/*!	\file dialogs/canvasresize.cpp
+**	\brief Template File
+**
+**	$Id$
+**
+**	\legal
+**	Copyright (c) 2002-2005 Robert B. Quattlebaum Jr., Adrian Bentley
+**
+**	This file is part of Synfig.
+**
+**	Synfig is free software: you can redistribute it and/or modify
+**	it under the terms of the GNU General Public License as published by
+**	the Free Software Foundation, either version 2 of the License, or
+**	(at your option) any later version.
+**
+**	Synfig is distributed in the hope that it will be useful,
+**	but WITHOUT ANY WARRANTY; without even the implied warranty of
+**	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+**	GNU General Public License for more details.
+**
+**	You should have received a copy of the GNU General Public License
+**	along with Synfig.  If not, see <https://www.gnu.org/licenses/>.
+**	\endlegal
+*/
+/* ========================================================================= */
+
+#ifdef USING_PCH
+#	include "pch.h"
+#else
+#ifdef HAVE_CONFIG_H
+#	include <config.h>
+#endif
+
+#include <ETL/handle>
+#include <dialogs/canvasproperties.h>
```

Comment:
Please add the prefix "<gui" and sort it alphabetically

Reply:
like this `<gui/dialogs/canvasproperties.h>` ?

Reply:
Saw your recent PR. Got it and on it :)